### PR TITLE
[Doc Issue][Update GCC note with LOB Apps][2880178]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a851a34f97b1360360c69ef9d5d87ad39989957e/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2f18358f-5a92-4a27-b99f-bde81ddebc6a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eac02b6103d99b1630bdc2ad51eed6738b7e583e/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4338a529-877c-40d1-a332-7f2e90cc83d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/91aa53cb810d2765e7a7995de1be1c06720d6a3f/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2bfe02f0-cc5f-4966-b3a7-08615c7e4970) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/66b98cc4665caa299b5b5b6e22abb3af494fa494/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ff212e0f-f6ee-43ea-9b64-78c59be34224) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Nivedipa-MSFT — DISMISSED** [review]

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 1]
```diff
@@ -72,6 +72,11 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 
 You must be familiar with the following product concepts to create custom meeting experiences with apps in Microsoft Teams.
 
+> [!IMPORTANT]
+>
+> * Currently, third-party and meeting line-of-business apps are available in Government Community Cloud (GCC) but aren't available for GCC-High and Department of Defense (DOD) tenants.
```
Comment (tacri) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:77]: maybe "third-party and line-of-business apps built for meetings and calls are available...

**Review by tacri — COMMENTED** [review]
Please check with Manasi as well, to make sure the message is accurate

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b45ff5f16cb61d8b3cb7d5940ba3608b0a1b6db8/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/341b1dfe-a72f-4803-88c4-d491e8c2d395) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d584cd243165f9e75e8010fe084c231a0ad9213b/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93e9987b-724b-4e45-b07e-1ce883aabbfe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-npaladugu** [comment]
@tacri request you to please review the docs

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1810a7bade0cf464ec66f5e93a9f8eb234b71729/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 95 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e312e231-0fdb-42ca-9e83-1d084e589988) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-npaladugu** [comment]
@tacri, gentle reminder. Request you to please review the docs.

**Comment by v-npaladugu** [comment]
Hi @tacri, 

Gentle reminder. 
Request you to please review the docs.

Cc: @surbhigupta12

**Comment by v-npaladugu** [comment]
Hi @tacri,

Gentle reminder.
Request you to please review the docs.

Cc: @surbhigupta12

**Comment by v-npaladugu** [comment]
Hi @tacri,

Gentle reminder.
Request you to please review the docs.

Cc: @surbhigupta12

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b46e25f52979958517d955b44689cbb8619915d2/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36957e4c-d0df-46e1-bd31-94aa7ce8a7c3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-npaladugu** [comment]
Hi @tacri,

Gentle reminder.
Request you to please review the docs.

Cc: @surbhigupta12

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/16c8c7c9f029aae67fc658237783e81ba05ae12c/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fcd2faa2-1057-4f78-8403-297ebff64506) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a6b8dfe182951f4f9b459af762df704ff5b90628/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94365e90-33e9-4295-910d-ab319d2a1779) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae026c4f5c3d71940f6264da1f5e963e6974dcba/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c174f503-569a-4259-a6fe-c836fcf104a0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d00121743d25aa5c22e85f4dc0eeb1108394677a/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dec41f50-45aa-435e-8e13-129405ed2cfb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75376cef1fb2112944b851a4cc508f071a6560a5/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/350a3014-4878-4ab9-a9c3-fd14ee955604) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**